assets/css/editor-style-classic.css

```diff
@@ -8,8 +8,8 @@
 
 /*
  * Chrome renders extra-wide &nbsp; characters for the Hoefler Text font.
- * This results in a jumping cursor when typing in both the Classic and block
- * editors. The following font-face override fixes the issue by manually
+ * This results in a jumping cursor when typing in both the classic editor and block
+ * editor. The following font-face override fixes the issue by manually
```

Comment:
The line length here exceeds 80 characters, can you adjust and word wrap to the next/following lines so that each remains less than 80 characters please